src/components/avatar/AvatarImage.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
   /** An alternative text for the image element. */
   imageAlt?: string;
   /** Callback function that is fired when user clicks the edit icon. */
-  onImageChange?: (event: React.MouseEvent<HTMLDivElement>) => void;
+  onImageChange?: (event: React.MouseEventHandler<HTMLDivElement>) => void;
```

Comment:
Did you mean to replace this with just `React.MouseEventHandler<HTMLDivElement>` ?

Reply:
Yeah i fixed it later on 👍

Reply:
Ah I missed that 🙈